cli/bpmetadata/proto/bpmetadata_ui.proto

```diff
@@ -148,11 +148,13 @@ message DisplayVariable {
   }
   repeated AlternateDefault alt_defaults = 21; // @gotags: json:"altDefaults,omitempty" yaml:"altDefaults,omitempty"
 
-  repeated DisplayVariableToggle toggle_using_variables = 22; // @gotags: json:"toggleUsingVariables,omitempty" yaml:"toggleUsingVariables,omitempty"
+  DisplayVariableToggle toggle_using_variable = 22;  // @gotags: json:"toggleUsingVariable,omitempty" yaml:"toggleUsingVariable,omitempty"
+
+  repeated DisplayVariableToggle toggle_using_variables = 23; // @gotags: json:"toggleUsingVariables,omitempty" yaml:"toggleUsingVariables,omitempty"
 
   // properties is a map defining all the fields of the input variable.
   // Gen: Manually-authored
-  map<string, DisplayVariable> properties = 23;
+  map<string, DisplayVariable> properties = 24;
```

Comment:
@apeabody @njlisi @g-awmalik Changing the field number should be fine as this proto is not yet used in g3. PTAL!